pkg/services/session/storage/persistent/executor.go

```diff
@@ -41,21 +48,15 @@ func (s *TokenStore) Create(ctx context.Context, body *session.CreateRequestBody
 	err = s.db.Update(func(tx *bbolt.Tx) error {
 		rootBucket := tx.Bucket(sessionsBucket)
 
-		ownerBucket, err := rootBucket.CreateBucketIfNotExists(ownerBytes)
+		ownerBucket, err := rootBucket.CreateBucketIfNotExists(id.WalletBytes())
```

Comment:
```suggestion
		userBucket, err := rootBucket.CreateBucketIfNotExists(id.WalletBytes())
```

if we want to force "owner" as the wrong naming